src/components/molecules/Visualizer/Engine/Cesium/hooks.ts

```diff
@@ -190,6 +196,14 @@ export default ({
     updateCamera();
   }, [updateCamera]);
 
+  // camera
```

Comment:
Don't feel like this helps understand the code more. If I look at the useEffects contents, or the code around the useEffect, I can see we are dealing with the camera. If you want to be more descriptive, it could help, but also the code inside the useEffect seems to be doing a camera flight based the emittedCamera, so nothing overly complex (unless the REASON for the flight is important, which might be worth making a comment about)

Reply:
This comment also comes from earlier code. I think i should remove it.